types/serverless/plugins/aws/provider/awsProvider.d.ts

```diff
@@ -384,6 +384,7 @@ declare namespace Aws {
         batchSize?: number | string;
         startingPosition?: number | string;
         enabled?: boolean;
+        type?: string;
```

Comment:
you can event more specific since only allowed types are `'dynamodb'` and `'kinesis'`
https://github.com/serverless/serverless/blob/b7901cdb77cb2c81dee62cb614d39d5d2fc824ff/lib/plugins/aws/package/compile/events/stream/index.js#L23
```suggestion
        type?: "dynamodb" | "kinesis";
```